### PR TITLE
bugfix for no tutorial courses

### PR DESCRIPTION
**Pull request by kenneth-miura**
Bug where it didn't generate schedules if one of the courses asked for had no tutorials.

### CODE REVIEW

**Review by roryhofland — APPROVED** [review]

**Review by EclipseIsDead — APPROVED** [review]
LGTM

**Review by lorena-b — APPROVED** [review]

File: src/main/java/workers/Scheduler.java [hunk 1]
```diff
@@ -103,6 +103,17 @@ public Schedule createBasicSchedule(ArrayList<Course> courses) {
         List<Section> courseLectures = c.getLectures();
         List<Section> courseTutorials = c.getTutorials();
 
+        if (courseTutorials.size() == 0){
```
Comment (sugolov) [inline src/main/java/workers/Scheduler.java:106]: we would need to add respective changes in populatePermutations and extendPermutations for this to affect permutationScheduler.

Comment (kenneth-miura) [reply src/main/java/workers/Scheduler.java:106]: ah yeaj good catch

Comment (kenneth-miura) [reply src/main/java/workers/Scheduler.java:106]: I just need to change extendPermutations right? then both populatPermutations & extendPermutations are altered and therefore permutationScheduler has the bug fixed

**Pull request merged**